tools/android/android_sdk_repository_template.bzl

```diff
@@ -159,6 +160,16 @@ def create_android_sdk_rules(
             tags = ["__ANDROID_RULES_MIGRATION__"],
         )
 
+        native.toolchain(
+            name = "sdk-%d-toolchain" % api_level,
+            toolchain_type = "@bazel_tools//tools/android:sdk_toolchain_type",
+            exec_compatible_with = HOST_CONSTRAINTS,
+            target_compatible_with = [
+                "@bazel_tools//platforms:android",
+            ],
+            toolchain = ":sdk-%d" % api_level,
```

Comment:
Will the toolchains also live in that same BUILD.tools ?

Reply:
No, this file is loaded from the BUILD file written by `android_sdk_repository`, so the toolchains are part of that repo.